src/deploy/database/prepare.ts

```diff
@@ -1,28 +1,27 @@
-"use strict";
+import * as _ from "lodash";
+import * as clc from "cli-color";
+import * as path from "path";
 
-var _ = require("lodash");
-var clc = require("cli-color");
-var path = require("path");
+import { FirebaseError } from "../../error";
+import * as parseBoltRules from "../../parseBoltRules";
+import * as rtdb from "../../rtdb";
+import * as utils from "../../utils";
+import { Options } from "../../options";
 
-var { FirebaseError } = require("../../error");
-var parseBoltRules = require("../../parseBoltRules");
-var rtdb = require("../../rtdb");
-var utils = require("../../utils");
+import * as dbRulesConfig from "../../database/rulesConfig";
 
-const dbRulesConfig = require("../../database/rulesConfig");
-
-module.exports = function (context, options) {
-  var rulesConfig = dbRulesConfig.getRulesConfig(context.projectId, options);
-  var next = Promise.resolve();
+export function prepare(context: any, options: Options): Promise<any> {
+  const rulesConfig = dbRulesConfig.getRulesConfig(context.projectId, options);
+  const next = Promise.resolve();
 
   if (!rulesConfig || rulesConfig.length === 0) {
     return next;
   }
 
-  var ruleFiles = {};
-  var deploys = [];
+  const ruleFiles: Record<any, any> = {};
```

Comment:
Is this really any, any? I think we can at least go Record<string,any>

Reply:
Sure.